### ISSUE TITLE
THIS IS A VIRUS: DO NOT DOWNLOAD 

### ISSUE THREAD

**Issue by kelten5784** [palahsu/DDoS-Ripper#27]
https://github.com/palahsu/DDoS-Ripper/files/8355861/dripper.txt

**Comment by mollthecoder** [comment]
How exactly is this malicious? Where is the malicious code?

**Comment by Zero23574** [comment]
brother i read the code theres nothing malicious about it

**Comment by notdvrk** [comment]
are u retarded?

**Comment by lucas-farias07** [comment]
# DO NOT USE www.development-tools.net
# IT'S A MALWARE

it is saying that development-tools.net is a malware

**Comment by mollthecoder** [comment]
> # DO NOT USE [www.development-tools.net](http://www.development-tools.net)
> # IT'S A MALWARE
> it is saying that development-tools.net is a malware

Yeah, but in the file attached there is no suspicious code as far as I can see. Correct me if I'm wrong.

**Comment by pavitra1129** [comment]
@mollthecoder  bro you are correct i have also reviewed the code but nothing suspicious

**Comment by Anonymous1503** [comment]
dumb shit

average script kiddie fr

**Comment by Andreesava** [comment]
lol it not a virus

**Comment by UnixCro** [comment]
Okay, so why don't they just write the normal code in the PR? 

And a cool program!
